package.json

```diff
@@ -49,8 +49,12 @@
     "@babel/core": "^7.5.5",
     "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
+    "@babel/runtime": "^7.23.9",
     "ganache": "7.3.1",
     "solc": "^0.4.26",
     "tape": "^5.6.1"
+  },
+  "peerDependencies": {
+    "@babel/runtime": "^7.21.0"
```

Comment:
I'm sure I'm missing something, but I'm not understanding why this needs to be here. Is it because `ethjs-query` has a peer dependency on Babel? If so, 1) it's not the same version range that `ethjs-query` is using and 2) why can't we remove that first?

Reply:
Okay, thanks for explaining. I'll make some time to modernize this package so we aren't using Babel to compile this package anymore (I've already made a couple of PRs).